README.md

```diff
@@ -52,6 +52,26 @@ Next follow these instructions.
 5. To run the migrations run: `python manage.py migrate`
 6. You can terminate the process by `Ctrl+C` in your terminal.
 
+### Run with Docker
+
+1. Make sure the latest version of docker-compose is installed by running ``docker-compose -v``. If it is not, install using ``pip install -U docker-compose``
+
+2. Change Database HOST in ``settings.py``
+
+- Go to ``main/settings.py``
+- Under databases change host to db.
+
+3. **Build docker image**
+
+Execute ``sudo docker-compose run web`` to build the image.
+
+4. **Run docker image**
+
+Run ``sudo docker-compose up`` from the root directory of project. Navigate to `http://localhost:8000` in your browser.
+
```

Comment:
There should be a command for migrating in docker. Just a suggestion :)

Reply:
@codesankalp the user can log into the container once build and run any command he wants.

`docker exec -it {container_name} /bin/bash` will open a terminal for the same.